packages/features/shell/Shell.tsx

```diff
@@ -1144,6 +1147,23 @@ export function ShellMain(props: LayoutProps) {
                 </div>
               )}
               {props.actions && props.actions}
+              {props.heading === "Bookings" && buttonToShow && (
+                <Button
+                  color="primary"
+                  onClick={buttonToShow === ButtonState.ALLOW ? enableNotifications : disableNotifications}
+                  loading={isLoading}
+                  disabled={buttonToShow === ButtonState.DENIED}
```

Comment:
Not part of this PR but I think the ButtonState enums are not consistent
```
export enum ButtonState {
  NONE = "none",
  ALLOW = "allow",
  DISABLE = "disable",
  DENIED = "denied",
}
``` 
in `package/lib/hooks`, we should probably update DENIED to DENY to be consistent. Again, super NIT that is not really part of this PR, but is related.